pkg/common/consumer/consumer_handler.go

```diff
@@ -29,6 +29,7 @@ type KafkaConsumerHandler interface {
 	// The returned error is enqueued in errors channel.
 	Handle(context context.Context, message *sarama.ConsumerMessage) (bool, error)
 	SetReady(ready bool)
+	GetConsumerGroup() string
```

Comment:
The way this interface evolved is a bit weird, but it's ok we can refactor later